### ISSUE TITLE
Add support for Danish Wikipedia?

### ISSUE THREAD

**Issue by fuzheado** [hatnote/listen-to-wikipedia#39]
Some how Danish is not in the UI, and adding "da" in the URL does not work. Can you add support for Danish Wikipedia changes feed? Thanks.

**Comment by zblace** [comment]
YES - I also tried adding HR (Croatian), BS (Bosnian) and SH (Serbo-Croatian), but did not get anything :-/

**Comment by marcolussetti** [comment]
The backend for it needs to be updated first for it to be supported.

**Comment by slaporte** [comment]
That's right, we'll need to set up [wikimon](https://github.com/hatnote/wikimon) on the backend to add a language. It's doable for these languages, although at some point I think we'll want to switch to SSE (per #31) to support all languages.
